src/app/reducers/index.ts

```diff
@@ -0,0 +1,19 @@
+import {
```

Comment:
need to create separate folder store
in this folder create folders for actions, selectors, effects etc.